screens/Playlist.js

```diff
@@ -166,18 +146,11 @@ const PlaylistScreen = ({ navigation, route }) => {
                       style={styles.thumbnailStyle}
                       resizeMode="cover"
                     />
-                    <View style={{ flex: 1, flexDirection: 'column' }}>
-                      <Heading style={styles.title}>{title}</Heading>
-                      <Text style={[styles.description, { paddingTop: 8 }]}>
-                        {new Date(publishDate).toLocaleDateString()}
-                      </Text>
```

Comment:
we probably don't _need_ to show the date...it was kinda more confusing than useful cause it's not necessarily when the message was given